addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -1732,6 +1733,36 @@ QUnit.test('first unseen message should be directly preceded by the new message
     );
 });
 
+QUnit.test('focus should be on textarea on manual send [REQUIRE FOCUS]', async function (assert) {
```

Comment:
'composer should be focused automatically after clicking on the send button [REQUIRE FOCUS]'